### PR TITLE
add obis-observer implementation

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: src/obis-observer/test.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
+/* eslint-disable */
```
Comment (DarkPark) [inline src/obis-observer/test.ts:1]: it's best to remove this file

File: src/utils/bitSet.ts [hunk 2]
```diff
@@ -61,3 +61,40 @@ export const toObject = ( bitMask: TBitSetMask = {}, value = 0 ): TBooleanObject
...
+/**
+ * Fill number of bits from position with number.
+ *
+ * @example
+ * ```js
+ * console.log(fillBits(0b11, 2, 3, 3));
+ * // 15 i.e. 0b1111
+ * ```
+ */
+export const fillBits = ( value: number, bitsNumber: number, startIndex: number, valueToSet:number ) => {
```
Comment (DarkPark) [inline src/utils/bitSet.ts:90]:
```suggestion
export const fillBits = ( value: number, bitsNumber: number, startIndex: number, valueToSet: number ) => {
```

File: src/obis-observer/Command.ts [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    size: unknown;
+
+
+    /** Get command ID in hex format. */
+    static getId () {
+        return `0x${this.id.toString(16).padStart(2, '0')}`;
+    }
+
+    /** Get command ID and title. */
+    static getName () {
```
Comment (DarkPark) [inline src/obis-observer/Command.ts:45]: should be removed

File: src/obis-observer/Command.ts [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+
+    parameters: unknown;
+
+    size: unknown;
+
+
+    /** Get command ID in hex format. */
+    static getId () {
+        return `0x${this.id.toString(16).padStart(2, '0')}`;
```
Comment (DarkPark) [inline src/obis-observer/Command.ts:41]: let's copy implementation from analog

**Review by DarkPark — CHANGES_REQUESTED** [review]

File: src/obis-observer/Command.ts (hunk 4 above)
Comment (DeadByElpy) [reply src/obis-observer/Command.ts:41]: fixed

File: src/utils/bitSet.ts (hunk 2 above)
Comment (DeadByElpy) [reply src/utils/bitSet.ts:90]: fixed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**